modules/SocketManager/module/auto/SocketManager.yml

```diff
@@ -60,6 +60,9 @@ cdefs: &cdefs
 - SOCKETMANAGER_CONFIG_MAX_SOCKETS:
     doc: "Maximum number of sockets supported"
     default: 1024
+- SOCKETMANAGER_CONFIG_MAX_POLL_TIMEOUT:
+    doc: "Restrict the poll time out (in ms) to improve responsiveness. -1 for infinite"
```

Comment:
How about ""Restrict the poll time out (in ms) to improve responsiveness; set to a negative value for infinite timeout" ?  It seems that poll only cares that the timeout is negative.

Reply:
Sure will update.